package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "@salesforce/plugin-code-analyzer",
-	"version": "5.0.0-beta.3",
+	"version": "5.0.0",
```

Comment:
Interesting! This just hadn't been done yet?

Reply:
That's a little odd, yeah. I'm not 100% positive it matters, though?